pages/common/sdk.md

```diff
@@ -0,0 +1,28 @@
+# sdk
+
+> Tool for managing parallel versions of multiple Software Development Kits.
+> Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
+
+- Install a specific version of Gradle:
+
+`sdk install gradle {{gradle_version}}`
```

Comment:
Is Gradle an example here (i.e. could it be `sdk install kotlin ...`)?

If so, `gradle` should be in token syntax for each example. 👍

Reply:
> Is Gradle an example here 
Correct.

Thanks, commands have been reformatted